tests/sentry/migrations/test_0816_update_notificationmessage_constraints_for_action_group_open_period.py

```diff
@@ -0,0 +1,242 @@
+from datetime import timedelta
+
+import pytest
+from django.db import IntegrityError
+from django.utils import timezone
+
+from sentry.models.rule import Rule
+from sentry.models.rulefirehistory import RuleFireHistory
+from sentry.testutils.cases import TestMigrations
+
+
+class TestAddActionColsToThreadsModel(TestMigrations):
```

Comment:
Rather than make this a migration test, which will roll the migration back and forward between each test, you could instead just make this a normal `TestCase` and mark it with `@pytest.mark.migrations`. This requires that the tests run with migrations, but doesn't roll things backwards and forwards.

There are a few examples of this in `getsentry`